README.md

```diff
@@ -38,12 +36,6 @@ To properly use Draft to containerize your application, create manifest files, a
 
```

Comment:
The Draft Create Deployment Files command isn't really a thing anymore.

Instead we have Draft a Dockerfile and Draft a Kubernetes Deployment and Service.